CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# 1.14.0-beta.1
+- Unlimited assets changes (#294)
```

Comment:
Maybe to match the previous format in the Changelog, we need a `## Changed` ? but I don't see too much value in it for one PR so I'm also ok with approving.

Reply:
Added the ##Changed title.  Good catch.

Reply:
Hmm, to be honest I would set this to `## Added` (without the trailing colon as well) as it's adding the unlimited assets support